src/openqdc/datasets/molecule3d.py

```diff
@@ -46,15 +46,19 @@ def _read_sdf(sdf_path, properties_path):
 
 class Molecule3D(BaseDataset):
     __name__ = "molecule3d"
-    __energy_methods__ = ["b3lyp_6-31g*"]
+    __energy_methods__ = ["b3lyp/6-31g*"]
+    # UNITS MOST LIKELY WRONG, MUST CHECK THEM MANUALLY
```

Comment:
@FNTwin is this already checked?

Reply:
No, I need to download the dataset and run some calculation and manually see what unit they are using as they don't provide any info. This PR should be just to have a quick method to interface openMLIP and convert the datasets between units for experimenting. A more heavy PR with every unit checked and validated will be with the calculated isolated atom energy